RSS Reader/RSS Reader/Topics list/View /Topic cell/RSSRTopicTableViewCell.m

```diff
@@ -0,0 +1,67 @@
+//
+//  RSSRTopicTableViewCell.m
+//  RSS Reader
+//
+//  Created by Lina Loyko on 11/17/20.
+//
+
+#import "RSSRTopicTableViewCell.h"
+#import "UIColor+RSSRColor.h"
+#import "NSDate+StringConversion.h"
+
+static NSString * const kTopicDateFormat = @"MMM d, yyyy HH:mm";
+static NSInteger const kCellViewCornerRadius = 10;
+
+@interface RSSRTopicTableViewCell ()
+
+@property (retain, nonatomic) IBOutlet UILabel *title;
+@property (retain, nonatomic) IBOutlet UILabel *pubDate;
+@property (retain, nonatomic) IBOutlet UILabel *summary;
+@property (retain, nonatomic) IBOutlet UIView *cellView;
+
+@end
+
+@implementation RSSRTopicTableViewCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    
+    self.cellView.layer.cornerRadius = kCellViewCornerRadius;
+    self.selectionStyle = UITableViewCellSelectionStyleNone;
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+    
+    if (selected) {
+        self.cellView.backgroundColor = UIColor.RSSRGray;
```

Comment:
This syntax violates Objective-C requirements. In case you are accessing a property, then you should use for syntax.In case you are calling a function you should use whitespace and square brackets.